ui/src/pages/version/components/forms/components/feast_config/components/FeastServingUrlSelect.js

```diff
@@ -15,10 +15,33 @@ const logoMaps = {
   bigtable: logoBigtable
 };
 
-export const FeastServingUrlSelect = ({ servingUrl, onChange, isInvalid }) => {
+export const FeastServingUrlSelect = ({
```

Comment:
Can you update the PR description with the screenshot of this change

Reply:
there is no change on UI look